napi/oxlint2/package.json

```diff
@@ -11,7 +11,7 @@
     "build-napi-test": "pnpm run build-napi --profile coverage --features force_test_reporter",
     "build-napi-release": "pnpm run build-napi --release",
     "build-js": "node scripts/build.js",
-    "test": "tsc && vitest"
+    "test": "tsc && vitest --dir ./test run"
```

Comment:
```suggestion
    "test": "tsc && vitest --dir ./test run",
    "test:watch": "vitest"

Reply:
I agree, watch mode is nice. But, unless I'm misunderstanding, when you update a file, tests will re-run, but `e2e` tests are still running against the *old* code because that's what `dist` still contains. So it's not really watching your changes at all.

Reply:
oh you're right

let's leave for now

Reply:
there probably is a vitest setting, but i think it's not a priority to make work rn